core/storage/user/gae_models.py

```diff
@@ -46,8 +46,12 @@ class UserSettingsModel(base_models.BaseModel):
 
     # User id used to identify user by GAE. Is not required for now because we
     # need to perform migration to fill this for existing users.
+    # Will be deprecated. This is moved to UserAuthModel now.
     gae_id = ndb.StringProperty(required=True, indexed=True)
-    # Email address of the user.
+    # Email address of the user. Won't be a required field in future. Because
```

Comment:
Do not add forward-looking comments in code docstrings. They tend to go stale and become wrong.

Instead, make sure your project is broken down into proper milestones such that each PR is self-contained and does not require reference to "this will be done in the future". The develop branch should be releasable at any time.